src/core/packages/logging/browser-internal/src/logger.ts

```diff
@@ -24,6 +25,7 @@ export class BaseLogger extends AbstractLogger {
     meta?: Meta
   ): LogRecord {
     if (isError(errorOrMessage)) {
+      apm.captureError(Object.assign({}, errorOrMessage, { meta, context: this.context, level }));
```

Comment:
While it would probably be a code smell to log an error on info level, maybe we should explicitly only report when `isError(errorOrMessage) && (level == 'error' || level == 'fatal')`

Reply:
since this is an internal tool, wouldn't it be best to capture any level and fix the code logging the error?

No strong opinion, though. Happy to add if clauses to limit the cases when we log them.